src/private/main.rs

```diff
@@ -107,8 +107,8 @@ fn main() {
     let mirror_api_client = {
         let env = Arc::new(grpcio::EnvBuilder::new().build());
         let ch = ChannelBuilder::new(env)
-            .max_receive_message_len(std::i32::MAX)
-            .max_send_message_len(std::i32::MAX)
+            .max_receive_message_len(-1)
```

Comment:
I think this should be unlimited already if it is set to usize::MAX. https://docs.rs/grpcio/0.2.3/grpcio/struct.ChannelBuilder.html#method.max_receive_message_len